docs/consensus/proof_of_archival_storage.md

```diff
@@ -297,33 +297,34 @@ Otherwise, submit a vote extrinsic with `solution`.
 4. Verify that `solution_range` and `global_randomness` are correct for the `slot_number` of the block.
 5. Compute the `global_challenge = hash(global_randomness||slot_number)`.
 6. Verify that current chain history size in segments is greater than winning `piece_index / NUM_PIECES`.
-7. Verify that `piece_offset ≤ max_pieces_in_sector`
-8. Re-derive `sector_id`
+7. Verify that the `history_size` in the solution is not greater than the current chain `history_size`.
```

Comment:
It'd make sense to mention edge-cases that I described in https://github.com/autonomys/subspace/pull/3632